proto/plan.proto

```diff
@@ -297,6 +297,8 @@ message TableDef {
 	string name				= 1;
 	repeated ColDef cols	= 2;
 	repeated DefType defs	= 3;
+	string TableType 		= 4;
```

Comment:
table_type  and create_sql are items of defs.
may be you don't need add these field in TableDef。
when you use, you can get them in TableDef.Defs